bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -127,6 +130,12 @@ for i in range(len(args.trigger_files)):
                             args.gating_veto_windows)
     ifo = reader.ifo
     trigs.ifos.append(ifo)
+    
+    # We don't have that many triggers, see if we can skip some templates
+    if len(reader.file[ifo]['template_id']) < 2**27:
+        uniq = numpy.unique(reader.file[ifo]['template_id'][:])
```

Comment:
How are the efficiency tests on this? - numpy.unique on (2 ** 27 - 1) templates feels like it will be slow

Apparently pandas.unique is a lot faster (O(N) rather than O(NlogN), but I've not checked that (https://stackoverflow.com/a/15637512)

Reply:
Seems to take just a few seconds when I run by hand, I think that's fast enough that it shouldn't be any significant impact to overall runtime.

Reply:
i.e. the worst case is that everything is unique. Seems good enough that performance is not a convern.

```
In [11]: %timeit numpy.unique(numpy.arange(0, 2**27, 1))
1.94 s ± 37.7 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```